geode-lucene/src/main/java/org/apache/geode/cache/lucene/internal/repository/IndexRepositoryImpl.java

```diff
@@ -95,6 +98,9 @@ public void update(Object key, Object value) throws IOException {
       docs.forEach(doc -> SerializerUtil.addKey(key, doc));
       Term keyTerm = SerializerUtil.toKeyTerm(key);
       writer.updateDocuments(keyTerm, docs);
+    } catch (Exception e) {
```

Comment:
Should this catch wrap writer.updateDocuments, or should it just wrap the call toDocuments?